io/key/key.go

```diff
@@ -64,6 +65,23 @@ type EditEvent struct {
 	Text string
 }
 
+// InputMode changes the on-screen-keyboard type. That hints the
+// type of data that might be entered by the user.
+type InputMode uint8
+
+const (
+	// KeyboardDefault is the default keyboard (alphanumeric).
+	KeyboardDefault InputMode = iota
```

Comment:
How about `HintAlphanumeric`, so that detail is moved from the comment to the name? If `alphanumeric` is too specific for you, I suggest `HintNone`.

Speaking of which, is there a need for both `HintNone` and `HintAlphanumeric`? The former represents no hints (TYPE_NULL in Android) while the latter explicitly asks for the alphanumeric input type.

Reply:
That makes sense. I also thought about others settings such as "autocapitalize", "grammar-correction" and "auto-complete", which could be disabled in some circumstances.

Since JS is the only OS that can support such feature, for now, maybe we can use as such:

- HintNone: Basic keyboard (without autocapitalize; without grammar-correction;  without autocomplete...). 
On Android it will use the TYPE_NULL.

- HintText: Alphanumeric keyboard (with all features). 
On Android it will use TYPE_TEXT (once it gets supported :S ).